insights/parsers/neutron_sriov_agent.py

```diff
@@ -0,0 +1,38 @@
+"""
+NeutronSriovAgent - file ``/etc/neutron/plugins/ml2/sriov_agent.ini``
+=====================================================================
+
+This class provides parsing for the files:
+    ``/etc/neutron/plugins/ml2/sriov_agent.ini``
+    ``/var/lib/config-data/puppet-generated/neutron/etc/neutron/plugins/ml2/sriov_agent.ini``
+
+Sample input data is in the format::
+
+    [DEFAULT]
+    debug = false
+
+    [sriov_nic]
+    physical_device_mappings=datacentre:enp2s0f6
+
+    [agent]
+    polling_interval=2
+
+    [securitygroup]
+    firewall_driver=noop
+    report_interval = 60
+
+    [keystone_authtoken]
+    auth_port = 35357
+
+See the ``IniConfigFile`` class for examples.
+"""
+from .. import IniConfigFile, parser, add_filter
+from insights.specs import Specs
+
+add_filter(Specs.neutron_sriov_agent, ["["])
+
+
+@parser(Specs.neutron_sriov_agent)
```

Comment:
This spec needs to be defined in the registry (`insights/specs/__init__.py`) and added to defaults (`insights/default.py`) in order for the data to be collected.